scripts/setup_local_network.sh

```
@@ -16,3 +16,22 @@ yarn run dashmate setup local --verbose \
+# local devnet bring-up. Production / release builds explicitly do NOT set this.
+#
+# CARGO_BUILD_PROFILE=release is mandatory at the current default N=1_000_000.
+# Release-mode Sinsemilla is ~10× faster than debug; without it the bake
+# stage during `docker build` would take hours. Apply at runtime InitChain
+# is ~134 ms regardless of profile (single SST ingest, no Sinsemilla work).
+#
+# Drop back to `dev` only if you also lower
+# `ShieldedSeedConfig::sdk_test_data().total_notes` to ~5k or below
+# (debug bake fits in tenderdash's InitChain window only at small N).
+for i in $(seq 1 ${MASTERNODES_COUNT}); do
+    yarn dashmate config set --config=local_${i} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)